sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/constant/Constants.java

```diff
@@ -68,6 +68,7 @@
     /**
      * Telnet Server
      */
+    public final static String TELNET_SERVER_ENABLE                  = "sofa.ark.telnet.server.enable";
     public final static String TELNET_PORT_ATTRIBUTE                 = "sofa.ark.telnet";
```

Comment:
How about "sofa.ark.telnet.server.port" ?  Is there a compatibility problem?

Reply:
yes. I plan to fix in next major version.